Updated feedback page (#5122)
PostHog/posthog.com: contents/handbook/product/user-feedback.md

### Feedback call process

**Recruiting users**

Ways to invite users for an interview:
- Using the [User Interview App](https://github.com/posthog/user-interview-app#adding-a-user-interview), see the [instructions](https://github.com/posthog/user-interview-app#adding-a-user-interview) for how to use it.
- If a customer has requested a feature through slack then message them directly
- Email customers who have subscribed to the feature on the [roadmap](https://posthog.com/roadmap)

**Scheduling**
- Add all feedback calls to the [User interviews calendar](https://calendar.google.com/calendar/?cid=Y19tczllaWN1Ym92ZGgxYWhzNmtoY2xpNTQ3b0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t). If the invite was created from your own calendar, you can simply add "User interviews" as an invitee.

**During the call**
- If you'd like to record the call, do ask before!
- Do a quick round of intros if you haven't met previously.
- If this is the first interview with the user, ask them for context about their company, their role, if they're technical.

**After the call**
1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.

> 2. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.

Reviewer: ```suggestion
2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel
3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
```

3. Generate the reward for the user (see below).

>    1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify. Make sure to **add a link to their Hubspot profile** in the notes section and set the **default expiration date** (5 years), and make note of the gift card code.

Reviewer: ```suggestion
   1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify.
```

Reviewer: @lharries I saw you have removed Hubspot further down, so we can remove it here as well.

4. Follow-up with the user. Send any applicable rewards, links to any opened GitHub issues, and answers to any outstanding questions.

### Rewards 
We strongly value our users' time. As such, we usually send a small gift of appreciation. We have the following general _guidelines_, but just use your best judgement.

> - Generally we send users a gift card to the merch store with around $30 of value.

Reviewer: @lharries You changed the amount from $50 to $30. I believe the user interview popup still says $50? If so, let me know once you've updated it and I know what to pay users before/after the change.

Author: I think $30 makes more sense? I'll update the user interview app

Author: Done

Reviewer: I think $30 is a good amount, especially with the options we have in the merch store 👍

**During the call**
- If you'd like to record the call, do ask before!
- Do a quick round of intros if you haven't met previously.
- If this is the first interview with the user, ask them for context about their company, their role, if they're technical.

**After the call**
1. Add the notes to the [Google Doc][feedback-doc], linking the recording if you did one.
2. Share a short summary of the user interview in the `#posthog-feedback` Slack channel
3. If the user reported specific bug reports or specific feature requests, open the relevant issues in GitHub. Be sure to link to their Hubspot record in case Engineering needs more context when scoping/building.
3. Generate the reward for the user (see below).
   1. Most of the time, the reward will be a gift card for the [PostHog merch store](https://merch.posthog.com/). If it's the case, create the gift card in Shopify.
4. Follow-up with the user. Send any applicable rewards, links to any opened GitHub issues, and answers to any outstanding questions.

### Rewards 
We strongly value our users' time. As such, we usually send a small gift of appreciation. We have the following general _guidelines_, but just use your best judgement.

> - Generally we send users a gift card to the merch store with around $30 of value.

Reviewer: ```suggestion
- Generally we send users a gift card to the [PostHog merch store](https://merch.posthog.com/) with around $30 of value.
```

> - When the above is not an option (e.g. shipping restrictions, user has received merch already) we default to Amazon gift cards. Use your Brex card to purchase these (and remember to follow our [spending guidelines](/handbook/people/spending-money)). Contact Paolo to join our Amazon Business account.

Reviewer: ```suggestion
- When the above is not an option (e.g. user has received merch already) we can offer the user a $30 [Open Collective](https://opencollective.com/) gift card.
- The instructions on how to create gift cards can be found [here](https://posthog.com/handbook/growth/sales/yc-onboarding#after-the-call)
```
